order/order.go

```diff
@@ -0,0 +1,119 @@
+package order
+
+import (
+	//	"fmt"
```

Comment:
goimports should help with this sort of thing

On Fri, Dec 18, 2015 at 7:44 AM, Evan J Johnson <notifications@github.com>
wrote:

> In order/order.go
> <https://github.com/cloudflare/redoctober/pull/106#discussion_r48037347>:
>
> > @@ -0,0 +1,119 @@
> > +package order
> > +
> > +import (
> > +	//	"fmt"
>
> my auto-gofmt is slacking here!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudflare/redoctober/pull/106/files#r48037347>.
>